web/templates/blocks/health_container.html.tmpl

```diff
@@ -1,15 +1,15 @@
 {{ define "health_container" }}
     <h5>Health</h5>
     <div class="health-container{{ if eq .Layout "horizontal" }} horizontal-container{{ end }}">
-        <div class="alert alert-inline alert-success">
+        <div class="alert alert-inline alert-success health-passing">
```

Comment:
I understand that we add this classes for DOM finding, etc.
But, is this the common practice? Are classes used here, instead of `id`s or `names`.
It might create some confusion in the moment to find which classes are real style classes and which not

Reply:
> But, is this the common practice?
>
Guess so, @dottorblaster what do you think?

We can argue whether we want to use css selectors or element data attributes (which might actually be better :thinking: )

Yet I went like this for simplicity.
What I can do is add a `tn-` prefix to identify those `tn-health-passing` (which I did somewhere else, but not here)

What do you think?